Clearing up command vs response in install instructions (#2694)
ros2/ros2_documentation: source/Installation/_Apt-Repositories.rst

.. code-block:: bash

   apt-cache policy | grep universe

> If you don't see an output line like the one below, then enable the Universe repository with the instructions in this section.

Reviewer: Personally I still find this a bit confusing.  I think it makes more sense to say something like:

```suggestion
This should output a line like the one below:
```

And then reinstate the sentence below.  That way it flows continuously from the beginning to the end.  Thoughts?
